README.md

```diff
@@ -75,21 +76,15 @@ prompt = Prompt(messages=[Message(content=prompt_template, role="user")])
 # Setup your LLM (Large Language Model) Node
 llm = OpenAI(
     id="openai",  # Unique identifier for the node
-    connection=OpenAIConnection(api_key="$OPENAI_API_KEY"),  # Connection using API key
+    connection=OpenAIConnection(api_key=os.getenv("OPENAI_API_KEY")),  # Connection using API key
```

Comment:
Why user should set the API key in env?
User can init connection just with `OpenAIConnection(api_key="user_api_key_value_here")`. Using env, read from file, API and so on based on where the user save API key.

Also `OpenAIConnection()` would try to automatically load API from `"OPENAI_API_KEY"` env if it not passed direclty.

So maybe just rename `"$OPENAI_API_KEY"` to `"OPENAI_API_KEY"` string or smth like this for a clear understanding that here user needs to pass API key